framework/src/play/src/main/scala/play/api/Configuration.scala

```diff
@@ -37,8 +41,9 @@ object Configuration {
         devSettings.get("config.file").orElse(Option(System.getProperty("config.file")))
           .map(f => new File(f)).getOrElse(new File(appPath, "conf/application.conf"))
       }
+      val playOverridesConfig: Config = getPlayOverridesConfig(Thread.currentThread().getContextClassLoader())
```

Comment:
If there is no reference-overrides in Play 2.3, why do we need to suppress the warnings, there should not be any warnings with akka default settings. I must be missing something.